mona_sdk/client.py

```diff
@@ -114,15 +115,20 @@
     "SAMPLING_FACTORS_MAX_AGE_SECONDS", 300
 )
 
-SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service."
+SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service"
 UPLOAD_CONFIG_ERROR_MESSAGE = (
-    "Could not upload the new configuration, please check it is valid."
+    "Could not upload the new configuration, please check it is valid. "
```

Comment:
```suggestion
    "Could not upload the new configuration, please check it is valid"
```
(for consistency - all end with periods or none)